docs/micro-lessons/196-lefthook-biome-quality-gates.md

```
@@ -0,0 +1,430 @@
+
+**Before**: Slow, manual hook setup
+```bash
+# Contributor workflow (OLD)
+git clone repo
+pnpm install
+# ... forgot to install hooks ...
+git push   # ❌ No validation! Manual setup needed
+```
+
+**Sequential pre-push** (15-30s):
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)